### PR TITLE
add infos about extensions from cn-tools

### PR DESCRIPTION
**Pull request by cn-tools**
It would be nice if you accept it.
It is promotion around my own extensions for FreshRSS only.

### CODE REVIEW

**Comment by Alkarex** [comment]
Ref: https://github.com/FreshRSS/FreshRSS/pull/2704

File: README.md [hunk 1]
```diff
@@ -43,3 +43,8 @@ There are some FreshRSS extensions out there, developed by community members:
 ### By [@Korbak](https://github.com/Korbak)
 
 * [Invidious](https://github.com/Korbak/freshrss-invidious): Displays videos from YouTube feeds inline and replaces every source by the Invidious instance of your choice for an enhanced privacy (no tracking or limitation)
+
+### By [@CN-Tools](https://github.com/cn-tools)
+
+* [YouTube Channel 2 RSSFeed](https://github.com/cn-tools/cntools_FreshRssExtensions/tree/master/xExtension-YouTubeChannel2RssFeed): You can add a YouTube Channel URL and will get it as RSSFeed
+* [Feed Title Builder](https://github.com/cn-tools/cntools_FreshRssExtensions/tree/master/xExtension-FeedTitleBuilder): Build your own feed title based on url, days date and original feed title
```
Comment (Frenzie) [inline README.md]: By "days date", do you mean "entry date"?

**Comment by cn-tools** [comment]
Dear @Frenzie

Sorry for my englisch :D
I mean the day when the user add the feed

**Comment by Frenzie** [comment]
The day the user added the feed? It's probably best to just say "the date the feed was added" then.

Is that something you refer to with `Tagesdatum`? (I noticed it says you're from Austria, so I figured it's probably some kind of Denglish.) I thought Tagesdatum was just today's date.

**Comment by cn-tools** [comment]
YES it was 'denglish'

i pull a new one #66

**Pull request closed without merging**